config/peerpods/podvm/osc-podvm-create-job.yaml

```diff
@@ -29,12 +31,11 @@ spec:
         - name: create
           # Binaries like kubectl, packer and yq are expected to be under /usr/local/bin
           # podvm binaries are expected to be under /payload/podvm-binaries.tar.gz
-          image: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-builder-rhel9:1.7.0
+          image: quay.io/suprit_iti/openshift-sandboxed-containers/osc-podvm-builder-rhel9:testv29
```

Comment:
Change the image source to registry.redhat.io and use it once the changes related to Dockerfile is approved